lca-cli/postpivot/postpivot.go

```diff
@@ -464,14 +460,6 @@ func (p *PostPivot) restoreOadpDataProtectionApplication(ctx context.Context, cl
 	return nil
 }
 
-func (p *PostPivot) copyClusterConfigFiles() error {
-	if err := utils.CopyFileIfExists(path.Join(p.workingDir, common.ClusterConfigDir, path.Base(common.CABundleFilePath)),
```

Comment:
I believe there will not be new ca bundle files but we still need create it as in IBI we not always build on precaching and i believe we still need to set trusted bundle as possibly it will be required to pull recert image in case it was not precached for some reason

Reply:
Right now `copyClusterConfigFiles` is only called [after](https://github.com/openshift-kni/lifecycle-agent/blob/f331b9d0d2154601f7799dc29660976a4599db1a/lca-cli/postpivot/postpivot.go#L126-L132) recert, is that a bug?